model/minsky.cc

```diff
@@ -1389,22 +1389,26 @@ namespace minsky
        {
          if (auto g=dynamic_cast<GodleyIcon*>(i->get()))
            {
+			 string newName;  
              if (type!=VariableType::flow)
                for (auto v: g->flowVars())
                  if (v->valueId()==name)
-                   throw error("flow variables in Godley tables cannot be converted to a different type");
+                   try {newName=v->name()+"^{Flow}"; g->table.renameFlows(v->name(),newName);}
```

Comment:
Current renameFlows does not throw, so this try/catch block will not catch anything.

Reply:
Don't you want to check somewhere that the new name is not an existing flow?